kythe/cxx/indexer/cxx/IndexerASTHooks.cc

```diff
@@ -587,10 +587,10 @@ const clang::Decl* IndexerASTVisitor::GetInfluencedDeclFromLValueHead(
   return nullptr;
 }
 
-const clang::Expr* IndexerASTVisitor::SkipTrivialAliasing(
-    const clang::Expr* expr) {
+std::pair<const clang::Expr*, std::optional<GraphObserver::NodeId>>
```

Comment:
std::pair has terrible ergonomics; prefer a named struct with semantically relevant member names: https://google.github.io/styleguide/cppguide.html#Structs_vs._Tuples

This is especially relevant here, where the second of the two returned values are not obvious and needs documenting.